aleks/resume.md

```diff
@@ -1,29 +1,46 @@
-## Résumé: Aleksandar Todorović 	
+# Résumé: Aleksandar Todorović 	
 
 ### Personal Details:
 
 #### Affiliation:
-Bits of Freedom, Open Web Fellow
+[Bits of Freedom](https://www.bof.nl/), Open Web Fellow
 
 #### Contact:
-* **Email**: [aleksandar@mozillafoundation.org](mailto:aleksandar@mozillafoundation.org)
-* **Twitter**: [@r3bl_](https://twitter.com/r3bl_)
-* **GitHub**: [@aleksandar-todorovic](https://github.com/aleksandar-todorovic)
 
-As part of the 2017 Open Web Fellows program, I completed/participated in the following:
+* **Emails**: [aleksandar@mozillafoundation.org](mailto:aleksandar@mozillafoundation.org) / [aleksandar@r3bl.me](mailto:aleksandar@r3bl.me) / [aleksandar.todorovic@bof.nl](mailto:aleksandar.todorovic@bof.nl)
```

Comment:
wow, why does bof include first AND last name, so many characters!?

Reply:
Wouldn't know why, you'd have to ask Hans. But yeah, my life would be a bit simpler if I didn't have to remember that 1/3 email addresses contains my last name as well.